### PR TITLE
chore: view other profile by username

### PR DESCRIPTION
**Pull request by nguyennh4522**
**What does this PR do?**

-   [x] chore: view other profile by username

**Media (Loom or gif)**
image
image

### CODE REVIEW

**Review by trkhoi — APPROVED** [review]

File: src/commands/profile/profile.ts [hunk 1]
```diff
@@ -416,63 +416,96 @@ const command: Command = {
...
+              page,
+            }),
+          }
+        },
+        false,
+        true
+      )
+    }
+
+    if (users.length == 0) {
```
Comment (anhnh12) [inline src/commands/profile/profile.ts:497]: what if user.length !== 0? return null?

Comment (nguyennh4522) [reply src/commands/profile/profile.ts:497]: show
https://user-images.githubusercontent.com/104887632/189586757-2b2a3aac-25da-4e67-a83f-c308e7a4e0e7.png

File: src/commands/profile/profile.ts [hunk 2]
```diff
@@ -416,63 +416,96 @@ const command: Command = {
...
-
-    listenForPaginateAction(
-      reply,
-      msg,
-      async (msg, page) => {
-        return {
-          messageOptions: await getCurrentViewEmbed({
+        .on("collect", async (i) => {
+          await i.deferUpdate()
+          if (i.customId === "selectView") {
```
[the remark this answers is not in the document]
Comment (nguyennh4522) [reply src/commands/profile/profile.ts:460]: I just move old code into a loop, i.values[0] contain different values for 2 options. I think we shouldn't use const here
image

File: src/commands/profile/profile.ts [hunk 3]
```diff
@@ -416,63 +416,96 @@ const command: Command = {
...
+            currentView = i.values[0]
+          } else if (i.customId == "selectCollection") {
+            currentCollection = i.values[0]
+          }
```
[the remark this answers is not in the document]
Comment (nguyennh4522) [reply src/commands/profile/profile.ts:464]: same reason above

File: src/commands/profile/profile.ts (hunk 1 above)
Comment (anhnh12) [reply src/commands/profile/profile.ts:497]: ===

**Pull request merged**